pkg/runtime/cli.go

```diff
@@ -55,6 +55,7 @@ func FromFlags() (*DaprRuntime, error) {
 	profilePort := flag.String("profile-port", strconv.Itoa(DefaultProfilePort), "The port for the profile server")
 	appProtocol := flag.String("app-protocol", string(HTTPProtocol), "Protocol for the application: grpc or http")
 	componentsPath := flag.String("components-path", "", "Path for components directory. If empty, components will not be loaded. Self-hosted mode only")
+	enableDynamicLoading := flag.Bool("enable-dynamic-loading", false, "Allow dynamic loading of components. This option is supported in self-hosted mode only")
```

Comment:
Can this be moved to the config CRD instead?

Reply:
switched to config feature instead of cli flag